pkg/reconciler/labeler/labelerv2_test.go

```diff
@@ -424,6 +441,28 @@ func patchAddLabel(namespace, name, key, value string) clientgotesting.PatchActi
 	return action
 }
 
+func patchAddRouteAndServingStateLabel(namespace, name, routeName string, now time.Time) clientgotesting.PatchActionImpl {
+	action := clientgotesting.PatchActionImpl{}
+	action.Name = name
+	action.Namespace = namespace
+
+	var state string
+	if routeName == "null" {
```

Comment:
`null` is a valid route name. I suggest you use a sentinel that is not a kubernetes identifier.

Reply:
this isn't intended as a sentinel value -  the test is looking at the raw json string. It should give

`serving.knative.dev/route: null`

rather than

`serving.knative.dev/route: "null"`

however it appears that the default json serializer isn't bothering with quotes on strings, so I'm kind of curious what would happen with a route named "null"

Reply:
https://golang.org/pkg/encoding/json/#Marshal

To confirm, the golang json marshal escapes the string value "null with quotes. All is well.

Reply:
Now it requires a comment, since others might wonder what would happen to a route named `null` 😆